tests/Tests.HubSpot/Deals/HubSpotDealConnectorTests.cs

```diff
@@ -97,10 +97,11 @@ public async Task GetAsync_returns_deal_with_pipeline_retrieved_by_http_client([
         }
 
         [Test, CustomAutoData]
-        public async Task GetAsync_returns_null_if_deal_retrieving_throws_NotFoundException([Frozen] IHubSpotClient hubSpotClient, [Frozen] IDealSelector dealSelector, HubSpotDealConnector sut)
+        public async Task GetAsync_returns_null_if_deal_retrieving_throws_NotFoundException([Frozen] IHubSpotClient hubSpotClient, [Frozen] IDealSelector dealSelector, HubSpotDealConnector sut, IFixture fixture)
         {
             //Arrange
-            Mock.Get(dealSelector).Setup(x => x.GetDeal(hubSpotClient)).Throws(new NotFoundException("Not Found", new Exception()));
+            var notFoundException = fixture.Create<NotFoundException>();
+            Mock.Get(dealSelector).Setup(x => x.GetDeal(hubSpotClient)).Throws(notFoundException);
 
```

Comment:
Instead of manually creating the NotFoundException using fixture (since no special customization is done), you can just accept it as a test method parameter, and let it be automatically created

Reply:
Indeed, that is a good point, fixed!